### PR TITLE
Clarify CreateTransaction error messages

### PR DESCRIPTION
**Pull request by instagibbs**
Logic change removing nValue check is fine since value won't be negative, and thus can give a better error.

### CODE REVIEW

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -2330,9 +2330,9 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
...
                         {
                             if (txout.nValue < 0)
-                                strFailReason = _("The transaction amount is too small to pay the fee");
+                                strFailReason = _("The transaction output amount is too small to pay the fee");
                             else
-                                strFailReason = _("The transaction amount is too small to send after the fee has been deducted");
+                                strFailReason = _("The transaction output amount is too small to send after the fee has been deducted");
                         }
                         else
                             strFailReason = _("Transaction amount too small");
```
Comment (morcos) [inline src/wallet/wallet.cpp]: Do you want to include this line too?
Not sure it's worth changing these, but if you are going to, perhaps you could be even more clear
"One of the transaction output amounts .." or similar

Comment (instagibbs) [inline src/wallet/wallet.cpp]: you're right not important, thought I changed it back

**Comment by sipa** [comment]
utACK

**Comment by laanwj** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/9204/commits/918b126901d6fd76f8ec49fce69a60d7819d3a57

**Comment by fanquake** [comment]
utACK 918b126

**Comment by jonasschnelli** [comment]
trivial utACK 918b126901d6fd76f8ec49fce69a60d7819d3a57

**Pull request merged**